docs/src/_contribute/ui.md

```diff
@@ -22,7 +22,7 @@ meltano init $PROJECT_NAME
 cd $PROJECT_NAME
 
 # Start the Meltano API and a production build of Meltano UI that you can ignore
-meltano ui
+FLASK_ENV=development meltano ui
```

Comment:
```suggestion
FLASK_DEBUG=True meltano ui
```

The `FLASK_ENV` setting has been [deprecated since Flask 2.2](
https://flask.palletsprojects.com/en/2.2.x/config/#ENV). We are using ^2.1, so `FLASK_DEBUG` should still be available, and setting it to true should accomplish what you want.

Reply:
If I don't set this flag (or `FLASK_DEBUG` as suggested, works as well), all requests from the dev UI are blocked because of a missing CORS header. So this is, afaict, a prerequisite to getting the dev UI up and running and as such I guess all contributors should use it.

Reply:
Seems it sets this header 
![image](https://user-images.githubusercontent.com/41419288/190394131-f9816deb-317a-41d0-8c13-21090289d5fd.png)